checkstyle-tester/projects-to-test-on-2.properties

```diff
@@ -0,0 +1,29 @@
+# List of GIT repositories to clone / pull for checking with Checkstyle
+# File format: REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+# Please note that bash comments works in this file
+
+azure-maven-plugins|git|https://github.com/microsoft/azure-maven-plugins|develop|
+pmd-master|git|https://github.com/pmd/pmd|master|
+eclipse-collections|git|https://github.com/eclipse/eclipse-collections|master|
+hadoop|git|https://github.com/apache/hadoop|trunk|
+intellij-plugins|git|https://github.com/JetBrains/intellij-plugins|master|
+flutter-intellij|git|https://github.com/flutter/flutter-intellij|master|
+jenkins|git|https://github.com/jenkinsci/jenkins|master|
+jacoco|git|https://github.com/jacoco/jacoco|master|
+junit4|git|https://github.com/junit-team/junit4|main|
+junit5|git|https://github.com/junit-team/junit5|main|
+kafka|git|https://github.com/apache/kafka|trunk|
+lucene|git|https://github.com/apache/lucene|main|
+maven|git|https://github.com/apache/maven|master|
+mssql-jdbc|git|https://github.com/microsoft/mssql-jdbc|main|
+OpenRefine|git|https://github.com/OpenRefine/OpenRefine|master|
+pitest|git|https://github.com/hcoles/pitest|master|
+tomcat|git|https://github.com/apache/tomcat|main|
+wildwebdeveloper|git|https://github.com/eclipse/wildwebdeveloper|master|
+mockito|git|https://github.com/mockito/mockito|main|
+processing|git|https://github.com/processing/processing|master|
+sonar-java|git|https://github.com/SonarSource/sonar-java|master|
+intellij-community|git|https://github.com/JetBrains/intellij-community|master|
+xwiki-platform|git|https://github.com/xwiki/xwiki-platform|master|
```

Comment:
We have xwiki in no-error mode in CI.
Let's not use them here. Better to choose another project.

Reply:
Done